Bottom: aa71a5694692dd9969e5d6fca99dc9517530f9f8
Top:    3b257643d41f65cf3e2d1573355a1bfdc1b1dfe5
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-04 10:56:18 -0500

Refresh of make

---

diff --git a/.farmrc b/.farmrc
index 2628b9a..ba873a1 100644
--- a/.farmrc
+++ b/.farmrc
@@ -4,6 +4,5 @@ v2.6.22 v2.6.23 v2.6.24 v2.6.25 v2.6.26 v2.6.27 v2.6.28 v2.6.29 v2.6.30
 v2.6.31 v2.6.32 v2.6.33 v2.6.34 v2.6.35 v2.6.36 v2.6.37 v2.6.38 v2.6.39
 v3.0 v3.1 v3.2 v3.3 v3.4 v3.5 v3.6 v3.7 v3.8 v3.9 v3.10 v3.11 v3.12
 linux-next/master"
-makeflags=-j11
 make="make -j11"
 export EXTRA_CFLAGS=-Werror-implicit-function-declaration
